e2e/testcafe-devextreme/tests/dataGrid/markup/T1240074_hoveringRows.ts

```diff
@@ -0,0 +1,27 @@
+import DataGrid from 'devextreme-testcafe-models/dataGrid';
+import { createWidget } from '../../../helpers/createWidget';
+import url from '../../../helpers/getPageUrl';
+import { getData } from '../helpers/generateDataSourceData';
+
+fixture.disablePageReloads`HoveringRows`.page(
+  url(__dirname, '../../container.html'),
+);
+
+test('Hover over a row in DataGrid', async (t) => {
+  const dataGrid = new DataGrid('#container');
+  const firstRow = dataGrid.getDataRow(0);
+
+  await t
+    .hover(firstRow.element)
+    .expect(firstRow.element.hasClass('dx-state-hover'))
+    .ok();
+}).before(async () => {
+  await createWidget(
+    'dxDataGrid',
+    {
+      dataSource: getData(5, 3),
+      hoverStateEnabled: true,
+    },
+    '#container',
```

Comment:
The third param of the ```createWidget``` functions is ```#container``` by default
[Link to the createWidget code](https://github.com/DevExpress/DevExtreme/blob/24_2/e2e/testcafe-devextreme/helpers/createWidget.ts#L39)

So, let's remove the third param :)
```suggestion
```